src/components/Card/card.jsx

```diff
@@ -25,6 +27,23 @@ export class Card extends Component {
     this.modifySmartLaunchUrls = this.modifySmartLaunchUrls.bind(this);
   }
 
+  takeSuggestion(suggestion, url) {
+    if (!this.props.isDemoCard) {
```

Comment:
Do we want to log something out to the console if its a Demo Card ?

Reply:
No, we should be ok here. This flag is just for the card demo to not actually execute a suggestion if a button is clicked on that demo view.